kernel/float64.ml

```diff
@@ -21,12 +21,17 @@ let is_neg_infinity f = f = neg_infinity
 
 (* Printing a binary64 float in 17 decimal places and parsing it again
    will yield the same float. We assume [to_string_raw] is not given a
-   [nan] as input. *)
+   [nan] or an infinity as input. *)
 let to_string_raw f = Printf.sprintf "%.17g" f
 
 (* OCaml gives a sign to nan values which should not be displayed as
    all NaNs are considered equal here *)
-let to_string f = if is_nan f then "nan" else to_string_raw f
+let to_string f =
+  if is_nan f then "nan"
+  else if is_infinity f then "infinity"
```

Comment:
Comment should be updated for the infinity printing
Something like `ocaml prints infinity as "inf" but we want "infinity"`

Reply:
Good idea.

BTW, do you think it is necessary to add an output test such as

```coq
Check 1e309%float.
Check (-1e309)%float.
```

?

Reply:
Good idea, done, thanks!